test/controllers/api/v1/session_controller_test.exs

```diff
@@ -0,0 +1,59 @@
+defmodule Habits.API.V1.SessionControllerTest do
+  use Habits.ConnCase
+
+  alias Habits.{Session, Account}
+  @valid_attrs %{email: "foo@bar.com", password: "p4ssw0rd"}
+
+  describe ".create" do
+
+    setup %{conn: conn} do
+      changeset = Account.changeset(%Account{}, @valid_attrs)
+      Repo.insert(changeset)
+      {:ok, conn: put_req_header(conn, "accept", "application/json")}
+    end
```

Comment:
I'm using a changeset here instead of a `Factory` function because I need the password properly encrypted.